apiserver/pom.xml

```diff
@@ -543,6 +537,13 @@
                         </property>
                     </systemProperties>
                 </configuration>
+                <dependencies>
+                    <dependency>
+                        <groupId>org.apache.maven.surefire</groupId>
+                        <artifactId>surefire-junit4</artifactId>
+                        <version>3.5.3</version>
+                    </dependency>
```

Comment:
Alpine v3.2.0 [already uses JUnit 5](https://github.com/stevespringett/Alpine/pull/609), but the API Server still requires JUnit 4. This dependency is necessary to ensure that Surefire can execute the JUnit 4 tests.